subjects/road_intersection/README.md

```diff
@@ -7,71 +7,106 @@ Its up to you to decide which library and file system you want do use to create
+#### **Environment and Rules**
+
+You must create an environment which contains all the objects described in this section. You can display the objects as you wish.
+
+1. Roads
+
+There exists various shapes of intersections, we will focus on the widely seen four-lane intersection. For simplicity each lane will have two directions.
 
 ```console
-                       |    ↓    |    ↑    |
-                       |    ↓    |    ↑    |
-                       |    ↓    |    ↑    |
-                       |    ↓    |    ↑    |
-                       |    ↓    |    ↑    |
-                       |         |    ↑    |
-                       |r0 r1 r2 |         |
-_______________________| ←  ↓  → |         |___________________
-                                 |            ↑ r3
-    ← ← ← ← ← ←                  |            ← r4    ← ← ← ← ←
-                                 |            ↓ r5
-_________________________________|______________________________
-                  r11 ↑          |
-   → → → → → →    r10 →          |                   → → → → → →
-                   r9 ↓          |
-_______________________          |          ____________________
-                       |         | ←  ↑  → |
-                       |    ↓    |r8 r7 r6 |
-                       |    ↓    |         |
-                       |    ↓    |    ↑    |
-                       |    ↓    |    ↑    |
-                       |    ↓    |    ↑    |
-                       |    ↓    |    ↑    |
-                       |    ↓    |         |
+               |    ↓    |    ↑    |
+               |    ↓    |    ↑    |
+               |    ↓    |    ↑    |
+               |    ↓    |    ↑    |
+               |    ↓    |    ↑    |
+               |r0 r1 r2 |    ↑    |
+_______________| ←  ↓  → |    ↑    |_____________
+                         |         ↑ r0
+← ← ← ← ← ← ←            |         ← r1 ← ← ← ← ←
+                         |         ↓ r2
+_________________________|_______________________
+           r2 ↑          |
+ → → → → → r1 →          |           → → → → → → 
+           r0 ↓          |
+_______________          |          _____________
+               |         | ←  ↑  → |
+               |    ↓    |r2 r1 r0 |
+               |    ↓    |    ↑    |
+               |    ↓    |    ↑    |
+               |    ↓    |    ↑    |
+               |    ↓    |    ↑    |
+               |    ↓    |    ↑    |
 ```
 
-There are 4 incoming lanes. Each lane has a set of consecutive unique ids `(ri, ri, ri)` where `(i = 0, 1, ⋯, 11)`, numbered clockwise starting from the top-most. Each set of ids
-indicates the outgoing direction of a vehicle. Vehicles driving in a given lane will have a given **route** (r0 | r1 .... r11) that indicates the lane destination of that vehicles.
+For clarification reasons we will assume that a lane can have three different routes (consider you are in the vehicle position):
+
+- `r0`, turning right
+- `r1`, straight ahead
+- `r2`, turning left
```

Comment:
What if we say `r`, for right. `l`, for left and `s` for straight? I think is a bit more clear

Reply:
ok ok done